x-pack/solutions/security/packages/features/src/attack_discovery/kibana_features.ts

```diff
@@ -9,9 +9,16 @@ import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
 import { i18n } from '@kbn/i18n';
 import { KibanaFeatureScope } from '@kbn/features-plugin/common';
 
-import { APP_ID, ATTACK_DISCOVERY_FEATURE_ID } from '../constants';
+import { APP_ID, ATTACK_DISCOVERY_FEATURE_ID, SERVER_APP_ID } from '../constants';
 import { type BaseKibanaFeatureConfig } from '../types';
 
+const alertingFeatures = [
+  {
+    ruleTypeId: 'attack-discovery',
```

Comment:
NIT: consider using a constant for this `ruleTypeId`

Reply:
It actually exists, just forgot to use it here. Thanks for catching this one!

Reply:
done in https://github.com/elastic/kibana/pull/216656/commits/bfb60dbc15491f1dd6fd7c28ee4eeeadd55e360f